tutorials/3d/environment_and_post_processing.rst

```diff
@@ -391,11 +598,149 @@ and Saturation:
+    when adjustments are enabled.
+
+Camera attribute options
+------------------------
+
+Depth of Field / Far Blur
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This effect simulates focal distance on cameras. It blurs objects behind
+a given range. It has an initial **Distance** with a **Transition** region
+(in world units):
+
+.. image:: img/environment_dof_far.png
+
+The **Amount** parameter controls the amount of blur. For larger blurs, tweaking
+the **Quality** may be needed in order to avoid artifacts.
+
+Depth of Field / Near Blur
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This effect simulates focal distance on cameras. It blurs objects close
+to the camera (acts in the opposite direction as far blur).
+It has an initial **Distance** with a **Transition** region (in world units):
+
+.. image:: img/environment_dof_near.png
+
+The **Amount** parameter controls the amount of blur. For larger blurs, tweaking
+the **Quality** may be needed in order to avoid artifacts.
+
+It is common to use both blurs together to focus the viewer's attention on a
+given object, or create a so-called
+`"tilt shift" effect <https://en.wikipedia.org/wiki/Miniature_faking>`__.
+
+.. image:: img/environment_mixed_blur.png
+
+Exposure
+^^^^^^^^
+
+Auto Exposure
+^^^^^^^^^^^^^
+
+*This feature is only available when using the GLES3 backend.*
+
+Even though, in most cases, lighting and texturing are heavily artist controlled,
+Godot supports a basic high dynamic range implementation with the auto exposure
+mechanism. This is generally used for the sake of realism when combining
+interior areas with low light and outdoors. Auto exposure simulates the camera
+(or eye) in an effort to adapt between light and dark locations and their
+different amounts of light.
+
+.. image:: img/environment_hdr_autoexp.gif
+
+The simplest way to use auto exposure is to make sure outdoor lights (or other
+strong lights) have energy beyond 1.0. This is done by tweaking their **Energy**
+multiplier (on the Light itself). To make it consistent, the **Sky** usually
+needs to use the energy multiplier too, to match with the directional light.
+Normally, values between 3.0 and 6.0 are enough to simulate indoor-outdoor conditions.
+
+By combining Auto Exposure with :ref:`doc_environment_and_post_processing_glow`
+post-processing, pixels that go over the tonemap **White** will bleed to the
+glow buffer, creating the typical bloom effect in photography.
+
+.. image:: img/environment_hdr_bloom.png
+
+The user-controllable values in the Auto Exposure section come with sensible
+defaults, but you can still tweak them:
+
+.. image:: img/environment_hdr.png
+
+- **Scale:** Value to scale the lighting. Brighter values produce brighter
+  images, smaller ones produce darker ones.
+- **Min Luma:** Minimum luminance that auto exposure will aim to adjust for.
+  Luminance is the average of the light in all the pixels of the screen.
+- **Max Luma:** Maximum luminance that auto exposure will aim to adjust for.
```

Comment:
These should be renamed given their respective names in CameraAttributesPhysical and CameraAttributesPractical (max_exposure_value/max_sensitivity)